src/backend/cpu/CMakeLists.txt

```diff
@@ -47,11 +47,24 @@ IF(NOT UNIX)
     ADD_DEFINITIONS(-DAFDLL)
 ENDIF()
 
+INCLUDE(ExternalProject)
+ExternalProject_Add(
+    threads
+    PREFIX ${CMAKE_BINARY_DIR}/third_party/threads
+    GIT_REPOSITORY https://github.com/alltheflops/threads.git
```

Comment:
Add a GIT_TAG. It can be a branch for the review process, but needs to be added.